tools/clang/test/HLSLFileCheck/pix/DebugHitGroupAndMiss.hlsl

```diff
@@ -0,0 +1,110 @@
+// RUN: %dxc -T lib_6_3 %s | %opt -S -hlsl-dxil-debug-instrumentation,parameter0=10,parameter1=20,parameter2=30 | %FileCheck %s
```

Comment:
Could we move this test to somewhere like tools/clang/test/PIX  to make it a lit FileCheck test?

Reply:
Does it make sense for me to move all the PIX tests at once?